sfepy/discrete/common/dof_info.py

```diff
@@ -460,6 +507,8 @@ def map_equations(self, bcs, field, ts, functions, problem=None,
         self.eq_ebc = nm.atleast_1d(ii.squeeze())
         self.val_ebc = nm.atleast_1d(val_ebc[ii].squeeze())
         self.master = nm.argwhere(master_slave > 0).squeeze()
+        # add axis in case we squeezed too hard
+        self.master = self.master if self.master.shape else self.master[None]
```

Comment:
When is this needed? Would using `nm.atleast_1d()` help as for `self.val_ebc`?

Reply:
This is needed in 1D problems for which line
```python
self.n_epbc = self.master.shape[0]
```
bellow would fail. However since DG is using dgepbc for 1D problems it is not necessary.